quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Utils/PlayerAction.java

```diff
@@ -27,7 +29,7 @@ public PlayerAction(int x, int y, boolean isHorizontal, boolean isFirst, Player
         this.isFirst = isFirst;
     }
 
-    public boolean isFirst() {
+    public boolean getIsFirst() {
```

Comment:
In java, boolean properties' getters are by convention start with "is" instead of "get".
It's bad practice to break standard convention.